### PR TITLE
added gino and aiocontextvars

### PR DESCRIPTION
**Pull request by Maxyme**

### CODE REVIEW

File: recipes/aiocontextvars/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools >=40.6.2
+    - pytest-runner >=4.2
+
+  run:
+    - python >=3.5
```
Comment (hmaarrfk) [inline recipes/aiocontextvars/meta.yaml]: I don't know if this selector is necessary.

Comment (hmaarrfk) [reply recipes/aiocontextvars/meta.yaml]: your right it is necessary.

**Comment by hmaarrfk** [comment]
@conda-forge/staged-recipes I think this package is ready.

File: recipes/aiocontextvars/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (hmaarrfk) [inline recipes/aiocontextvars/meta.yaml]: nit: I think the flavour of the week is to use `{{ PYTHON }}`

File: recipes/gino/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (hmaarrfk) [inline recipes/gino/meta.yaml]: nit: I think the flavour of the week is to use `{{ PYTHON }}`

File: recipes/aiocontextvars/meta.yaml (hunk 2 above)
Comment (beckermr) [reply recipes/aiocontextvars/meta.yaml]: Yes. This should be changed.

File: recipes/gino/meta.yaml (hunk 3 above)
Comment (beckermr) [reply recipes/gino/meta.yaml]: Ditto here.

**Comment by jakirkham** [comment]
Looks like the user deleted their fork. Going ahead and closing.

**Pull request closed without merging**